t/ui/18-tests-details.t

```diff
@@ -541,7 +541,7 @@ subtest 'additional investigation notes provided on new failed' => sub {
     $driver->get('/tests/99947');
     $driver->find_element('#clones a')->click;
     $driver->find_element_by_link_text('Investigation')->click;
-    ok($driver->find_element('#investigation_status_entry')->text_like(qr/No result dir/),
+    ok($driver->find_element('table#investigation_status_entry')->text_like(qr/No result dir/),
```

Comment:
An ID is already as specific/unique as it gets. It is a bit weird to combine it with further specifications like element names.

Reply:
well I added the check to make sure we are actually looking for a table element. Should I remove this again?

Reply:
Ah, ok - that makes sense.